crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -266,19 +267,35 @@ def repos_path
       Pathname.new(File.join("/srv/tftpboot", @platform, @arch, "repos"))
     end
 
+    def repo_path
+      repos_path.join(name)
+    end
+
+    def smt_path
+      unless @config["smt_path"].blank?
```

Comment:
Sure, but then I have a line that is too long. And your suggestion is 5 lines instead of 3 lines (for a readability that is equivalent :-)